main.go

```diff
@@ -63,14 +63,7 @@ func main() {
 	}
 
 	// Expose Vault token metrics.
-	{
-		if tokenPath == "" {
-			panic(microerror.Maskf(invalidConfigError, "path to token folder can not be empty"))
-		}
-		if vaultURL == "" {
-			panic(microerror.Maskf(invalidConfigError, "Vault URL can not be empty"))
-		}
-
+	if tokenPath != "" && vaultURL != "" {
```

Comment:
We don't want to run the vault token collector when on a tenant cluster, so we switch here now.

Reply:
That is ugly.

Reply:
Plan is to move this out to a separate exporter we only run on the control plane.